lib/jenkins_pipeline_builder/extensions/builders.rb

```diff
@@ -381,7 +381,7 @@
   builder do
```

Comment:
Actually amazingly you can nest these `builders` infinitely within each other and it just works <b><sup>TM</sup></b>